### ISSUE TITLE
Pipeline controller cloning step procedure

### ISSUE THREAD

**Issue by valeriano-manassero** [allegroai/clearml#272]
Hi,
during some tests I noticed Pipeline controller cloning steps is taking in consideration archived task also.

This results in cloned tasks that are already in archived state.
Moreover it drives to a lot of confusions with warnings since archived steps may be more than one
```
WARNING - Additionally found task
```

I think archived task should not be used at all from a Pipeline controller.

wdyt?

**Comment by bmartinn** [comment]
Hi @valeriano-manassero 
The pipeline itself is using the `Task.get_task` if a task ID is not provided (i.e. looking based on task project/name).
The `Task.get_task` will sort all the tasks based on task updated field, but will always put the archived tasks last, so if we are selecting the first one (i.e. the most updated one), we will not get the archived one.
1. I think we should probably remove the warnings as it is somewhat confusing.
2. I wouldn't want to actively block you from creating a pipeline on an archived Task, as it will make it very easy to break pipelines ... That said a warning if it was actually selected makes sense to me, WDYT?

BTW: regardless I would recommend to use he Task id not the Task name (which you can always change). To get the Task ID just press the ID button next to the Task name on the right info panel :)

**Comment by bmartinn** [comment]
@valeriano-manassero the bug has been fixed in v0.17.4
``` bash
pip install clearml==0.17.4
```

**Comment by valeriano-manassero** [comment]
I can confirm this is now ok.
